.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,78 @@
+## Description
+
+Fixes #.
+
+Changes proposed in this pull request:
+
+## Type of change
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Vulnerabilities update
+- [ ] Config changes
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires an admin update after deploy
+- [ ] Includes a database migration removing  or renaming a field
+
+
+## Testing
+
+How should the reviewer test this PR?
+Write out any special testing steps here.
+
+### Post-deployment actions
+
+In case this PR needs any admin changes or run a management command after deployment, mention it here:
+
+## Checklist
+
+### General checks
+
+- [ ] Linting and tests pass locally
+- [ ] The website and the changes are functional in Tor Browser
```

Comment:
Do we generally run PR tests in Tor Browser? I feel like I haven't really seen us do that so far.

Reply:
I always try to test any frontend changes in Tor Browser, especially if we are progressively enhancing something to add javascript functionality, to check how things work without JS.